tasks/main.yml

```diff
@@ -1,15 +1,15 @@
 ---
 - name: Get tarfile
-  command: wget https://github.com/kr/beanstalkd/archive/v1.9.tar.gz -q -O /tmp/beanstalkd.tar.gz creates=/tmp/beanstalkd.tar.gz
+  command: wget https://github.com/kr/beanstalkd/archive/v{{ beanstalkd_version }}.tar.gz -q -O /tmp/beanstalkd.tar.gz creates=/tmp/beanstalkd.tar.gz
 
 - name: Unarchive
-  command: tar -zxvf /tmp/beanstalkd.tar.gz chdir=/tmp creates=/tmp/beanstalkd-1.9
+  command: tar -zxvf /tmp/beanstalkd.tar.gz chdir=/tmp creates=/tmp/beanstalkd-{{ beanstalkd_version }}
 
 - name: Make
-  command: make chdir=/tmp/beanstalkd-1.9/ creates=/tmp/beanstalkd-1.9/beanstalkd
+  command: make chdir=/tmp/beanstalkd-{{ beanstalkd_version}}/ creates=/tmp/beanstalkd-{{ beanstalkd_version}}/beanstalkd
 
 - name: Install
-  command: make install chdir=/tmp/beanstalkd-1.9/ creates=/usr/local/bin/beanstalkd
+  command: make install chdir=/tmp/beanstalkd-{{ beanstalkd_version}}/ creates=/usr/local/bin/beanstalkd
 
```

Comment:
Good point.  Simply removing the `creates` seems ugly.   I'll give it some thought later and see if I can come up with a reasonable fix.